src/common/commonutils/PerfUtils.c

```diff
@@ -119,3 +119,19 @@ void LogPerfClock(PERF_CLOCK* clock, const char* componentName, const char* obje
         }
     }
 }
+
+void LogPerfClockTelemetry(PERF_CLOCK* clock, const char* targetName, const char* componentName, const char* objectName, int objectResult, OSCONFIG_LOG_HANDLE log)
+{
+    long microseconds = -1;
+
+    if (NULL == clock)
+    {
+        OsConfigLogError(log, "LogPerfClockTelemetry called with an invalid clock argument");
+        return;
+    }
+
+    microseconds = GetPerfClockTime(clock, log);
+
+    OsConfigLogInfo(log, "{\"TargetName\": \"%s\", \"ComponentName\": \"%s\", \"ObjectName\": \"%s\", \"ObjectResult\": \"%s (%d)\", \"Microseconds\": \"%ld\"}",
```

Comment:
nit: Could remove the extra spaces after `:` and `,` in JSON to reduce the size a bit. We do this with PowerShell a lot using the `-Compress` switch. I believe parson would do the same for single line serialization.

Reply:
Think prefer like this for cleanliness. This is not for users, is for the Telemetry parser that will come next.

Reply:
Did the change, looks better. Thanks